Aug 30 21:55:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306232]: Traceback (most recent call last):
Aug 30 21:55:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306232]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306232]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306232]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[306259]: Traceback (most recent call last):
Aug 30 21:55:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[306259]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[306259]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:03 ubuntu-s-1vcpu-1gb-sgp1-01 python[306259]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306263]: Traceback (most recent call last):
Aug 30 21:55:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306263]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306263]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306263]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306266]: Traceback (most recent call last):
Aug 30 21:55:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306266]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306266]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306266]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[306269]: Traceback (most recent call last):
Aug 30 21:55:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[306269]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[306269]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:12 ubuntu-s-1vcpu-1gb-sgp1-01 python[306269]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306274]: Traceback (most recent call last):
Aug 30 21:55:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306274]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306274]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306274]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306277]: Traceback (most recent call last):
Aug 30 21:55:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306277]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306277]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306277]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[306280]: Traceback (most recent call last):
Aug 30 21:55:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[306280]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[306280]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:20 ubuntu-s-1vcpu-1gb-sgp1-01 python[306280]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306282]: Traceback (most recent call last):
Aug 30 21:55:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306282]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306282]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306282]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[306285]: Traceback (most recent call last):
Aug 30 21:55:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[306285]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[306285]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:25 ubuntu-s-1vcpu-1gb-sgp1-01 python[306285]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[306287]: Traceback (most recent call last):
Aug 30 21:55:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[306287]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[306287]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:28 ubuntu-s-1vcpu-1gb-sgp1-01 python[306287]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[306289]: Traceback (most recent call last):
Aug 30 21:55:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[306289]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[306289]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:30 ubuntu-s-1vcpu-1gb-sgp1-01 python[306289]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[306294]: Traceback (most recent call last):
Aug 30 21:55:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[306294]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[306294]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:33 ubuntu-s-1vcpu-1gb-sgp1-01 python[306294]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[306296]: Traceback (most recent call last):
Aug 30 21:55:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[306296]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[306296]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:36 ubuntu-s-1vcpu-1gb-sgp1-01 python[306296]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[306298]: Traceback (most recent call last):
Aug 30 21:55:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[306298]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[306298]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:38 ubuntu-s-1vcpu-1gb-sgp1-01 python[306298]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[306302]: Traceback (most recent call last):
Aug 30 21:55:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[306302]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[306302]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:41 ubuntu-s-1vcpu-1gb-sgp1-01 python[306302]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[306304]: Traceback (most recent call last):
Aug 30 21:55:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[306304]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[306304]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:44 ubuntu-s-1vcpu-1gb-sgp1-01 python[306304]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[306307]: Traceback (most recent call last):
Aug 30 21:55:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[306307]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[306307]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:46 ubuntu-s-1vcpu-1gb-sgp1-01 python[306307]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[306311]: Traceback (most recent call last):
Aug 30 21:55:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[306311]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[306311]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:49 ubuntu-s-1vcpu-1gb-sgp1-01 python[306311]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[306312]: Traceback (most recent call last):
Aug 30 21:55:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[306312]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[306312]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:51 ubuntu-s-1vcpu-1gb-sgp1-01 python[306312]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:54 ubuntu-s-1vcpu-1gb-sgp1-01 python[306314]: Traceback (most recent call last):
Aug 30 21:55:54 ubuntu-s-1vcpu-1gb-sgp1-01 python[306314]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:54 ubuntu-s-1vcpu-1gb-sgp1-01 python[306314]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:54 ubuntu-s-1vcpu-1gb-sgp1-01 python[306314]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[306317]: Traceback (most recent call last):
Aug 30 21:55:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[306317]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[306317]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:56 ubuntu-s-1vcpu-1gb-sgp1-01 python[306317]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:55:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[306319]: Traceback (most recent call last):
Aug 30 21:55:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[306319]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:55:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[306319]:     from ai_supervisor import ask_with_routing
Aug 30 21:55:59 ubuntu-s-1vcpu-1gb-sgp1-01 python[306319]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306321]: Traceback (most recent call last):
Aug 30 21:56:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306321]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306321]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:01 ubuntu-s-1vcpu-1gb-sgp1-01 python[306321]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[306325]: Traceback (most recent call last):
Aug 30 21:56:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[306325]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[306325]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:04 ubuntu-s-1vcpu-1gb-sgp1-01 python[306325]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306328]: Traceback (most recent call last):
Aug 30 21:56:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306328]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306328]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:06 ubuntu-s-1vcpu-1gb-sgp1-01 python[306328]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306330]: Traceback (most recent call last):
Aug 30 21:56:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306330]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306330]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:09 ubuntu-s-1vcpu-1gb-sgp1-01 python[306330]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[306333]: Traceback (most recent call last):
Aug 30 21:56:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[306333]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[306333]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:11 ubuntu-s-1vcpu-1gb-sgp1-01 python[306333]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306337]: Traceback (most recent call last):
Aug 30 21:56:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306337]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306337]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:14 ubuntu-s-1vcpu-1gb-sgp1-01 python[306337]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306339]: Traceback (most recent call last):
Aug 30 21:56:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306339]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306339]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:17 ubuntu-s-1vcpu-1gb-sgp1-01 python[306339]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[306346]: Traceback (most recent call last):
Aug 30 21:56:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[306346]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[306346]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:19 ubuntu-s-1vcpu-1gb-sgp1-01 python[306346]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306350]: Traceback (most recent call last):
Aug 30 21:56:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306350]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306350]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:22 ubuntu-s-1vcpu-1gb-sgp1-01 python[306350]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[306352]: Traceback (most recent call last):
Aug 30 21:56:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[306352]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[306352]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:24 ubuntu-s-1vcpu-1gb-sgp1-01 python[306352]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[306355]: Traceback (most recent call last):
Aug 30 21:56:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[306355]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[306355]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:27 ubuntu-s-1vcpu-1gb-sgp1-01 python[306355]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:29 ubuntu-s-1vcpu-1gb-sgp1-01 python[306359]: Traceback (most recent call last):
Aug 30 21:56:29 ubuntu-s-1vcpu-1gb-sgp1-01 python[306359]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:29 ubuntu-s-1vcpu-1gb-sgp1-01 python[306359]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:29 ubuntu-s-1vcpu-1gb-sgp1-01 python[306359]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[306361]: Traceback (most recent call last):
Aug 30 21:56:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[306361]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[306361]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:32 ubuntu-s-1vcpu-1gb-sgp1-01 python[306361]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:34 ubuntu-s-1vcpu-1gb-sgp1-01 python[306365]: Traceback (most recent call last):
Aug 30 21:56:34 ubuntu-s-1vcpu-1gb-sgp1-01 python[306365]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:34 ubuntu-s-1vcpu-1gb-sgp1-01 python[306365]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:34 ubuntu-s-1vcpu-1gb-sgp1-01 python[306365]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:37 ubuntu-s-1vcpu-1gb-sgp1-01 python[306366]: Traceback (most recent call last):
Aug 30 21:56:37 ubuntu-s-1vcpu-1gb-sgp1-01 python[306366]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:37 ubuntu-s-1vcpu-1gb-sgp1-01 python[306366]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:37 ubuntu-s-1vcpu-1gb-sgp1-01 python[306366]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[306368]: Traceback (most recent call last):
Aug 30 21:56:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[306368]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[306368]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:40 ubuntu-s-1vcpu-1gb-sgp1-01 python[306368]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:42 ubuntu-s-1vcpu-1gb-sgp1-01 python[306374]: Traceback (most recent call last):
Aug 30 21:56:42 ubuntu-s-1vcpu-1gb-sgp1-01 python[306374]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:42 ubuntu-s-1vcpu-1gb-sgp1-01 python[306374]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:42 ubuntu-s-1vcpu-1gb-sgp1-01 python[306374]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[306435]: Traceback (most recent call last):
Aug 30 21:56:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[306435]:   File "/home/uaft/uaft_project_50lite/tg_runner.py", line 8, in <module>
Aug 30 21:56:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[306435]:     from ai_supervisor import ask_with_routing
Aug 30 21:56:45 ubuntu-s-1vcpu-1gb-sgp1-01 python[306435]: ImportError: cannot import name 'ask_with_routing' from 'ai_supervisor' (/home/uaft/uaft_project_50lite/ai_supervisor.py)
Aug 30 21:56:47 ubuntu-s-1vcpu-1gb-sgp1-01 python[306473]: INFO:aiogram:Bot: UAFT_BOT [@UAFT_bot]
Aug 30 21:56:47 ubuntu-s-1vcpu-1gb-sgp1-01 python[306473]: WARNING:aiogram:Updates were skipped successfully.
Aug 30 21:56:47 ubuntu-s-1vcpu-1gb-sgp1-01 python[306473]: INFO:aiogram.dispatcher.dispatcher:Start polling.
